.pre-commit-config.yaml

```diff
@@ -21,10 +22,8 @@ repos:
     rev: 5.6.4
     hooks:
     -   id: isort
-        name: isort (python)
-    -   id: isort
-        name: isort (cython)
-        types: [cython]
+        types: [text]
```

Comment:
would also be good add a comment here, like
```
        types: [text]  # overwrite upstream `types: [python]`
```
I think such comments work in yaml

Reply:
Added!